config/betamocks/services_config.yml

```diff
@@ -244,6 +244,16 @@
     :cache_multiple_responses:
       :uid_location: header
       :uid_locator: 'va_eauth_pnid'
+  
+  # VSO Search
+  - :method: :post
+    :path: <%= "/wss-common-services-web-#{Settings.evss.versions.common}/rest/vsoSearch/11.0/getCurrentInfo" %>
+    :file_path: "evss/vso_search/get_current_info"
+    :cache_multiple_responses:
+      :uid_location: header
```

Comment:
@bastosmichael should these be listed twice?

Reply:
So there are two ways to get at the data and since we are only working with mocked data I want to keep both in here until we know which one is actually going to work with staging.